charger/webasto.go

```diff
@@ -209,7 +209,7 @@ func (wb *WebastoLive) TotalEnergy() (float64, error) {
 		return 0, err
 	}
 
-	return float64(binary.BigEndian.Uint32(b)) * 100, nil
+	return float64(binary.BigEndian.Uint32(b)) / 1e3, nil
```

Comment:
Findest Du das in der Doku? Ich sehe da / 10.

Reply:
Ich da nach der Anzeige im Screenshot vorgegangen… 🤔